core/organization/service.go

```diff
@@ -29,19 +29,21 @@ type AuthnService interface {
 }
 
 type Service struct {
-	repository      Repository
-	relationService RelationService
-	userService     UserService
-	authnService    AuthnService
+	repository           Repository
+	relationService      RelationService
+	userService          UserService
+	authnService         AuthnService
+	orgDisableOnCreation bool
```

Comment:
we can create a variable to store `defaultState` as enabled/disabled instead and keep the if statement in constructor.